packages/fyllut-backend/src/config/config.ts

```diff
@@ -113,7 +113,7 @@ const config: ConfigType = {
   featureToggles: featureUtils.toFeatureToggles(process.env.ENABLED_FEATURES ?? defaultEnabledFeatures),
   isDevelopment: process.env.NODE_ENV === 'development',
   isTest: process.env.NODE_ENV === 'test',
-  isProduction: process.env.NODE_ENV === 'production',
+  isProduction: process.env.NODE_ENV === 'production' && process.env.NAIS_CLUSTER_NAME === 'prod-gcp',
```

Comment:
Ta utgangspunkt i naisClusterName der sjekken skjer i stedet for å endre isProduction